lib/gui/components/drive-selector/templates/drive-selector-modal.tpl.html

```diff
@@ -45,7 +50,15 @@ <h4 class="list-group-item-heading">{{ drive.description }} - {{ drive.size | gi
 </div>
 
 <div class="modal-footer">
+  <span ng-show="modal.state.getDrive().system" class="modal-footer-warning">
+      Etcher has identified this as a system drive
+  </span>
   <button class="button button-primary button-block"
+    ng-class="{ 'label-danger': modal.state.getDrive().system, 'button-primary': !modal.state.getDrive().system }"
```

Comment:
We should be re-using the `isSystemDrive()` function you added to `SelectionStateModel` instead of `modal.state.getDrive().system`.